src/main/java/org/sagebionetworks/web/client/view/SearchViewImpl.java

```diff
@@ -147,7 +147,7 @@ public void setSearchResults(SearchResults searchResults, String searchTerm) {
 		String facetNames = createShownFacets(searchResults);
 		String pageTitleSearchTerm = searchTerm != null && searchTerm.length() > 0 ? "'" + searchTerm + "' " : "";
 		synapseJSNIUtils.setPageTitle("Search: " + pageTitleSearchTerm + facetNames);
-		narrowResultsPanel.setVisible(true);
+		narrowResultsPanel.setVisible(searchResults.getFacets() != null);

```

Comment:
Only show the "Narrow Results" panel if the search results have facets.

Reply:
Is it always true that if this is not null, then it is also not empty?

Reply:
I'm assuming this is a list or some other collection

Reply:
Based on testing, this was always null when empty.  But I added an empty check for good measure